packages/documentation/library/pages/components/nav.md

```diff
@@ -0,0 +1,12 @@
+---
+title: Nav
+description: Documentation and examples of how to use NELSON navs
+---
+
+# Overview
+
+<framework-tabs></framework-tabs>
+
+#### Storybook
```

Comment:
also we wont be using storybook in the docs, it will be up to the developer to create the documentation and storybook will purely be used for developers to test the code.

Reply:
The files left in component are the generic component docs. The docs from each individual component are merged in and injected into the 'framework-tabs' tag.

The content of the generic component docs is just what was there before. It should be updated in a different ticket/pr.